src/sentry/analytics/events/ai_autofix_pr_events.py

```diff
@@ -0,0 +1,30 @@
+from sentry import analytics
+
+
+class AiAutofixPrEvent(analytics.Event):
+    type = "ai.autofix.pr.closed"
+
+    attributes = (
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("project_id"),
+        analytics.Attribute("group_id"),
+        analytics.Attribute("run_id"),
+        analytics.Attribute("integration"),
+    )
+
+
+class AiAutofixPrClosedEvent(AiAutofixPrEvent):
+    type = "ai.autofix.pr.closed"
```

Comment:
this is the same event as the parent, was this intended?

Reply:
whoops